cea/functions.py

```diff
@@ -2,13 +2,14 @@
 from __future__ import division
 
 import math
-import numpy as np
 import os
+
+import numpy as np
 import pandas as pd
 import scipy
 import scipy.optimize as sopt
-import sandbox.ghapple.reporting as reporting
 
+import contributions.reporting as reporting
```

Comment:
please do not import from contributions or sandbox in core! due to the way the python packaging system works, these create a lot of headache...